Modules/Microsoft365DSC/DSCResources/MSFT_EXOMailbox/MSFT_EXOMailbox.schema.mof

```diff
@@ -0,0 +1,171 @@
+[ClassVersion("1.0.0.0"), FriendlyName("EXOMailbox")]
+class MSFT_EXOMailbox : OMI_BaseResource
+{
+    [Key, Description("The Identity parameter specifies the Mailbox that you want to modify.")] String Identity;
+    [Write, Description("Mailbox cannot be created/removed in O365.  This must be set to 'Present'"), ValueMap{"Present"}, Values{"Present"}] String Ensure;
+    [Write, Description("The Alias parameter specifies the Exchange alias (also known as the mail nickname) for the recipient.")] String Alias;
+    [Write, Description("The EmailAddresses parameter specifies all the email addresses (proxy addresses) for the Mailbox")] string EmailAddresses[];
+    [Write, Description("The DisplayName parameter specifies the display name of the mailbox.")] String DisplayName;
+    [Write, Description("")] String DistinguishedName;
+    [Write, Description("")] String ExchangeVersion;
+    [Write, Description("")] String ExternalDirectoryObjectId;
+    [Write, Description("")] String Guid;
```

Comment:
If this is only read-only, let's not include it. Also, what is the difference between id and GUID?